0-ad-history-mod-package/simulation/templates/mixins/encyclopedia/encyclopedia_siege_lithobolos.xml

```diff
@@ -2,10 +2,9 @@
 <Entity>
   <Identity>
     <History>
-        Although the earliest mentions of stone throwing machines reach back much earlier, the first Greek catapult (Lithobolos) was developed in the early 4th century B.C. during Dionysius of Syracuse's expedition against the Carthaginians.
-        Unlike many weapons at the time, the lithobolos was powered by a torsion mechanism. Torsion-based weapons can store and release energy more efficiently than tension-based weapons (like bows and crossbows).
-        Compared to the oxybeles (Greek bolt shooter), the lithobolos was bigger, heavier, and more powerful and just had one single arm facing forward. One end of the arm was attached to the torsion mechanism. As the arm was rotated backwards, a bundle of sinew, skein, or fiber was twisted. A projectile was loaded and the lithobolos was fired. When the arm was released, the bundle unwound, launching the projectile at a 45-degree angle upward. The lithobolos could throw 150-pound stones hundreds of yards away. It was used to damage defensive structures such as walls or towers.
-        Because catapults were so bulky, they were carried in pieces and assembled on the battlefield.
+        The lithobolos (translated “stone-thrower”) was a type of Greek catapult probably first constructed for Dionysius of Syracuse's expedition against Carthage. Its defining feature was a newly developed mechanism: the lithobolos got its power from the torsion of bundles of sinew, skein, or fiber instead of the tension of a bow like traditional weapons did. The twisting was done by the catapult's arms running through the bundle and rotating as the bowstring was pulled back. For the latter, a winch mechanism was used. 
```

Comment:
```suggestion
        The lithobolos (translated “stone-thrower”) was a type of Greek catapult, probably first constructed for Dionysius of Syracuse's expedition against Carthage. Its defining feature was a newly developed mechanism: the lithobolos got its power from the torsion of bundles of sinew, skein, or fiber instead of the tension of a bow like traditional weapons did. The twisting was done by the catapult's arms running through the bundle and rotating as the bowstring was pulled back. For the latter, a winch mechanism was used. 
```